src/main/scala/org/dberg/hubot/listeners/TestListener3.scala

```diff
@@ -0,0 +1,21 @@
+package org.dberg.hubot.listeners
+
+import org.dberg.hubot.Hubot
+import org.dberg.hubot.listeners.Listener.CallbackSuccess
+import org.dberg.hubot.models._
+
+class TestListener3(hubot: Hubot) extends Listener(hubot) {
+
+  val callback: Callback = {
```

Comment:
The user would then be able to return CallbackSkipped, CallbackMiddlewareFailure, and CallbackNotMatched, which are a larger scope of failures than a listener cares about.